pages/prerequisites/content-information.md

```diff
@@ -17,33 +17,28 @@ Action Tips | 313 KB | 174 KB | 890 KB | N/A | N/A
-The Team Sessions | 140 MB | 119 MB | 162 MB | from 30 to 45 mn | 2000kB/s (peak), 400kB/s (average)
 
+### Mobile for CrossKnowledge content
 
-### Mobile for CrossKnowledge contents
-
-The following content compatible with iPad (2, 3, 4 and mini) - additional requirements may be needed for the content hosted on the Learning Suite.
+The following content compatible with iPad (latest version) - additional requirements may be needed for the content hosted on the Learning Suite.
 
 Compatibility with Android tablets is made but not officially supported.
 
 Format | Tablets | Smartphones
 ---- | ---- | ----
-Videocasts | iOS 4+: English + subtitles in 16 langues, Android 3.0+ : English, Mp4 format | iOS 4+: English, Android 4.0+, Mp4 format
-Essentials | iOS 4+ & Android 4.0+, For both, 15 languages, PDF format | iOS 4+ & Android 4.0+, For both, 15 languages, PDF format
-Action tips | iOS 4+ & Android 4.0+, For both, 16 languages, PDF format | iOS 4+ & Android 4.0+, For both, 16 languages, PDF format
-Business reviews | iOS 4+ & Android 4.0+, For both, 2 languages, PDF format | iOS 4+ & Android 4.0+, For both, 2 languages, PDF format
-The Team (LO Code TTxxxxx) and New CrossKnowledge Sessions (v3) | See Content produced with Mohive in Multi-devices Mode (HTML5)  | See Content produced with Mohive in Multi-devices Mode (HTML5)
-
+Videocasts | iOS (latest version): English + subtitles in 16 langues, Android (latest version) : English, Mp4 format | iOS (latest version): English, Android (latest version), Mp4 format
+Essentials | iOS (latest version) & Android (latest version), For both, 15 languages, PDF format | iOS (latest version) & Android (latest version), For both, 15 languages, PDF format
+Action tips | iOS (latest version) & Android (latest version), For both, 16 languages, PDF format | iOS (latest version) & Android (latest version), For both, 16 languages, PDF format
+Business reviews | iOS (latest version) & Android (latest version), For both, 2 languages, PDF format | iOS (latest version) & Android (latest version), For both, 2 languages, PDF format
 
-### About The Team v2 (LO code like TTxx3xx) and New CrossKnowledge Sessions
+### About New CrossKnowledge Sessions
 
-- Only deployable using CrossKnowledge Suite 12.1 (and newer versions)
-- The basic pre-requisites of CrossKnowledge Suite 12.1 (and newer versions) have to be applied plus: 600 kbit / s bandwidth (equivalent to 3G)
+- Only deployable using CrossKnowledge Suite 14.1+
+- The basic pre-requisites of CrossKnowledge Suite 14.1+ have to be applied plus: 600 kbit / s bandwidth (equivalent to 3G)
 - Cloud Mode: HTML5
-- Classic Mode: Flash Player 10
-- Microsoft Internet Explorer 8 minimum (popup blockers must be disabled or else allow the browser to accept pop-ups from CrossKnowledge.com).
+- Microsoft Edge latest version (popup blockers must be disabled or else allow the browser to accept pop-ups from CrossKnowledge.com).

```

Comment:
Flash player reference should be removed in next paragraph
flash player is dead so we should not support it anymore
althought I guess that some LO are still using it but it would be difficult to audit
https://www.adobe.com/products/flashplayer/end-of-life.html

Reply:
Ok I've removed all about Flash (player).